File: Examen_Unidad2/src/Director.java
import java.util.*;
public class Director {
        private static final String CONTRASENA_DIRECTOR = "123456789@";

        public static boolean autenticarDirector() {
            Scanner scanner = new Scanner(System.in);
            System.out.println("Ingrese la contraseña del director:");
            String input = scanner.nextLine();
            return input.equals(CONTRASENA_DIRECTOR);
        }
    }

File: Examen_Unidad2py/ZOO PY/Director.py
class Director:
  CONTRASENA_DIRECTOR = "123456789@" #contraseña asignada

  @staticmethod #declarar un método estático dentro de una clase
  #metodo
  def autenticarDirector():
      print("Ingrese la contraseña del director:")
      input_password = input()
      return input_password == Director.CONTRASENA_DIRECTOR

# Ejemplo
# if __name__ == "__main__":
#    if Director.autenticarDirector():
#        print("Contraseña correcta. Acceso concedido.")
#    else:
#       print("Contraseña incorrecta. Acceso denegado.")